Appsilon/rhino
# feat: Add lint_sass()

### Changes
Closes #28 

### How to test
Initialize a project and run `rhino::lint_sass()`. After installing Node, tests should pass. Try adding this to `app/styles/main.scss`:
```sass
h3 {
  font-size: 23px
}
```
Test should fail.

Note: The initialized project will pull `rhino` from your cache, which means that you can get the old version. Make sure that you have installed the newest one inside the project.
